mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/WEB-INF/src/com/liferay/todo/TodoPortlet.java

```diff
@@ -92,6 +93,28 @@ public void doView(
 			super.doView(renderRequest, renderResponse);
 	}
 
+	@Override
+	public void serveResource(
+			ResourceRequest resourceRequest, ResourceResponse resourceResponse)
+		throws IOException, PortletException {
+
+			String taskInput = ParamUtil.getString(
```

Comment:
```Input``` suffix is a convention for JS variable that holds an input node. Parameter name should reflect business meaning of the parameter, for example ```taskTitle``` or ```taskContent```.